examples/autoconf/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+	id 'java'
+	id 'application'
+	id 'com.google.cloud.tools.jib'
+}
+
+
+description = 'Examples for using java autoconfiguration and Google Cloud Operations'
+
+dependencies {
+	compile(libraries.opentelemetry_api)
+	compile(libraries.opentelemetry_api_metrics)
+	compile(libraries.opentelemetry_sdk_metrics)
+	runtimeOnly(libraries.opentelemetry_sdk_autoconf)
+	runtimeOnly(libraries.opentelemetry_sdk)
+	runtimeOnly project(':exporter-auto')
+	runtimeOnly project(':detector-resources')
+}
+
+mainClassName = 'com.google.cloud.opentelemetry.example.autoconf.AutoconfExample'
+
+application {
+	applicationDefaultJvmArgs = [
+		'-Dotel.traces.exporter=google_cloud_trace',
+		'-Dotel.metrics.exporter=google_cloud_monitoring',
+	]
+}
+
+
+jib {
+	from.image = 'gcr.io/distroless/java-debian10:11'
+	containerizingMode = 'packaged'
+	// TODO: Figure out how to pull versions from build.
+	container.jvmFlags = [
```

Comment:
Can this be pulled out into a constant instead of duplicating?

Reply:
done